website/source/docs/configuration/seal/azurekeyvault.html.md

```diff
@@ -71,9 +71,15 @@ Azure authentication values:
 * `AZURE_ENVIRONMENT`
 ```
 
-Note: If Vault is hosted on Azure, Vault can use Managed Service Identities (MSI) to access Azure instead of an environment and 
-shared client id and secret.  MSI must be [enabled](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/qs-configure-portal-windows-vm) 
-on the VMs hosting Vault. 
+~> **Note:** If Vault is hosted on Azure, Vault can use Managed Service
+Identities (MSI) to access Azure instead of an environment and  shared client id
```

Comment:
Two spaces between "and" & "shared"

Reply:
Good eyes. Thank you for catching this!  [Fixed](https://github.com/hashicorp/vault/pull/7414/commits/c9a0b33279c520e0eb7da269a802ac5775eaf9cf).